Sources/TuistSupport/Errors/FatalError.swift

```diff
@@ -34,7 +34,9 @@ public struct UnhandledError: FatalError {
         """
         We received an error that we couldn't handle:
             - Localized description: \(error.localizedDescription)
-            - Error: \(error)"
+            - Error: \(error)
+
+        If you think it's a legit issue, please file an issue including the reproducible steps: https://github.com/tuist/tuist/issues/new
```

Comment:
Shouldn't the link point to https://github.com/tuist/tuist/issues/new?assignees=&labels=&template=Bug_report.md ?

Reply:
That's the first idea that I had, but then I thought about it further and thought the new issue would be a better idea because it's very unlikely that URL changes, and I think it's straightforward for a user to pick the right template from the list. This way we have the flexibility to change the template names without breaking the contract between the CLI and the URL.

Reply:
Makes sense, then the link should be https://github.com/tuist/tuist/issues/new/choose (with `/choose` at the end) isn't it?

Reply:
You are right. I updated it to be the one that you suggested.